.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  semi: true,
+  trailingComma: "all",
+  singleQuote: true,
+  tabWidth: 2
```

Comment:
I think the only custom option is `singleQuote: true`?
```suggestion
  singleQuote: true,
```

Reply:
Looked at the options over at https://prettier.io/docs/en/options.html and found that `trailingComma` is also set to `es5` by default. I'll remove the other ones, though!